### DISCUSSION TITLE
Good number of "sets" to train to.

### DISCUSSION THREAD

**Discussion by rohskopf** [deepmodeling/deepmd-kit#217] [Q&A] [answered]
I'm training a potential for a single system, with a few different sets inside. Does it make a difference whether or not I split this system into many different sets, or just a few small number of sets? What effect does this have on training?

**Comment by jameswind** [comment, marked as answer]
For each system, the last set is used as testing data. If there is only one
set, it will purely be used for training. There are only small effects like
how data is loaded and randomly picked in the sgd optimization process.

**Reply by dkussainova** [reply]
If there is only one set and it is purely used for training, what is used for testing and how is data selected? I have only one data set, and I can see learning curves both for training and testing sets.
